nntrainer/tensor/uint_tensor.cpp

```diff
@@ -141,6 +148,25 @@ template <typename T> void *UIntTensor<T>::getData(size_t idx) const {
   return data->getAddr<T>() + offset + idx;
 }
 
+template <typename T> void *UIntTensor<T>::getScale() const {
+  if (!data)
+    return nullptr;
+
+  data->validate();
+  return ((T *)getData()) + size();
+}
+
+template <typename T> void *UIntTensor<T>::getScale(size_t idx) const {
+  NNTR_THROW_IF(idx > scale_size(), std::invalid_argument)
+    << "Tensor::getScale() index is not valid";
+
+  if (!data)
+    return nullptr;
+
+  data->validate();
+  return (T *)getData() + size() + idx;
```

Comment:
Is the type of scale `T`? AFIK, the type of scale is float; thus it should be updated as :
```suggestion
  return (float*)((T *)getData() + size()) + idx;
```

Reply:
You're absolutely correct! Thank you for pointing that out.